modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/wal/filehandle/FileWriteHandleImpl.java

```diff
@@ -476,8 +476,6 @@ private static void fsync(MappedByteBuffer buf, int off, int len) throws IgniteC
                     if (segRecPtr != null) {
                         FileWALPointer filePtr = (FileWALPointer)segRecPtr;
 
-                        fsync(filePtr);
```

Comment:
I think you can return these bad boys back

Reply:
Yep, forgot about this.